oxidation/crates/parsec_api_crypto/src/rustcrypto/constant.rs

```diff
@@ -0,0 +1,5 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+pub const SIGNATURE_LENGTH: usize = ed25519_dalek::SIGNATURE_LENGTH;
```

Comment:
Can you put this as `PublicKey::SIGNATURE_SIZE` to be coherent with the other constants ?

Reply:
Do you have a rational for replacing the `<Struct>::<constant>` style by this constant module ? (genuine question here 😄 )

Reply:
done under the name of `SIGNATURE_LENGTH`

Reply:
done in 0bf07c4